Buildings/Fluid/Chillers/ModularReversible/BaseClasses/RefrigerantCycleHeatRecovery.mo

```diff
@@ -0,0 +1,159 @@
+          -58,8}},
+      color={0,0,127}));
+  connect(refCycChiCoo.QCon_flow, swiQCon.u1)
+    annotation (Line(points={{26.6667,38.3333},{26.6667,8},{58,8}},color={0,0,127}));
+  connect(refCycChiCoo.PEle, swiPEle.u1)
+    annotation (Line(points={{40,38.3333},{40,-48},{8,-48},{8,-58}},color={0,0,127}));
+  connect(refCycChiCoo.QCon_flow, swiQCon.u3)
+    annotation (Line(points={{26.6667,38.3333},{26.6667,-8},{58,-8}},color={0,0,127}));
+  connect(refCycChiCoo.QEva_flow, swiQEva.u3)
+    annotation (Line(points={{53.3333,38.3333},{54,38.3333},{54,24},{-10,24},{
+          -10,-8},{-58,-8}},
+      color={0,0,127}));
+  connect(refCycChiCoo.PEle, swiPEle.u3)
+    annotation (Line(points={{40,38.3333},{40,-48},{-8,-48},{-8,-58}},color={0,0,127}));
+  annotation (
+    Icon(
+      coordinateSystem(
+        preserveAspectRatio=false),
+      graphics={
+        Rectangle(
+          extent={{-100,100},{100,-100}},
+          lineColor={238,46,47},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-24,88},{22,44}},
+          lineColor={28,108,200},
+          lineThickness=0.5),
+        Line(
+          points={{-16,82},{20,74}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{-18,52},{20,58}},
+          color={28,108,200},
+          thickness=0.5),
+        Rectangle(
+          extent={{-98,40},{-60,-28}},
+          lineColor={28,108,200},
+          lineThickness=0.5),
+        Line(
+          points={{-20,-60},{-20,-70},{-20,-80},{20,-60},{20,-80},{-20,-60}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{-122,34},{-66,34},{-82,10},{-66,-22},{-120,-22}},
+          color={28,108,200},
+          thickness=0.5),
+        Rectangle(
+          extent={{60,40},{98,-28}},
+          lineColor={28,108,200},
+          lineThickness=0.5),
+        Line(
+          points={{120,34},{64,34},{80,10},{64,-22},{118,-22}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{-80,40},{-80,68},{-24,68}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{22,66},{80,66},{80,40}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{78,-28},{78,-70}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{78,-70},{62,-70},{20,-70}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{-80,-26},{-80,-68},{-20,-68}},
+          color={28,108,200},
+          thickness=0.5),
+        Text(
+          extent={{-151,147},{149,107}},
+          textColor={0,0,255},
+          fillPattern=FillPattern.HorizontalCylinder,
+          fillColor={0,127,255},
+          textString="%name")}),
+    Diagram(
+      coordinateSystem(
+        preserveAspectRatio=false)),
+    Documentation(
+      revisions="<html>
```

Comment:
This is a new model. We would need to have the new revision note.

Reply:
Addressed